specification/devtestlabs/resource-manager/Microsoft.DevTestLab/2016-05-15/DTL.json

```diff
@@ -1,8584 +1,8658 @@
-                    "200": {
-                        "description": "OK",
-                        "schema": { "$ref": "#/definitions/NotificationChannel" }
-                    },
-                    "201": {
-                        "description": "Created",
-                        "schema": { "$ref": "#/definitions/NotificationChannel" }
-                    },
-                    "default": {
-                        "description": "BadRequest",
-                        "schema": { "$ref": "#/definitions/CloudError" }
-                    }
-                },
-                "security": [ { "oauth2": [ "user_impersonation" ] } ]
-            },
-            "delete": {
-                "tags": [ "NotificationChannels" ],
-                "description": "Delete notificationchannel.",
-                "operationId": "NotificationChannels_Delete",
-                "parameters": [
-                    { "$ref": "#/parameters/subscriptionId" },
-                    { "$ref": "#/parameters/resourceGroupName" },
-                    {
-                        "name": "labName",
-                        "in": "path",
-                        "description": "The name of the lab.",
-                        "required": true,
-                        "type": "string"
-                    },
-                    {
-                        "name": "name",
-                        "in": "path",
-                        "description": "The name of the notificationChannel.",
-                        "required": true,
-                        "type": "string"
-                    },
-                    { "$ref": "#/parameters/api-version" }
-                ],
-                "responses": {
-                    "200": { "description": "OK" },
-                    "204": { "description": "No Content" },
-                    "default": {
-                        "description": "BadRequest",
-                        "schema": { "$ref": "#/definitions/CloudError" }
-                    }
-                },
-                "security": [ { "oauth2": [ "user_impersonation" ] } ]
-            },
-            "patch": {
-                "tags": [ "NotificationChannels" ],
-                "description": "Modify properties of notificationchannels.",
-                "operationId": "NotificationChannels_Update",
-                "parameters": [
-                    { "$ref": "#/parameters/subscriptionId" },
-                    { "$ref": "#/parameters/resourceGroupName" },
-                    {
-                        "name": "labName",
-                        "in": "path",
-                        "description": "The name of the lab.",
-                        "required": true,
-                        "type": "string"
-                    },
-                    {
-                        "name": "name",
-                        "in": "path",
-                        "description": "The name of the notificationChannel.",
-               
```

Comment:
It seems`Operations_List` is missing?

Reply:
corrosponing [linter error](https://travis-ci.org/Azure/azure-rest-api-specs/jobs/285852306#L575)

Reply:
Talked to @ajay-arora about this today, he clarified that this end-point has nothing to with the `ARM Operations API` that returns all the operations supported by the RP.

The endpoint `/subscriptions/{subscriptionId}/providers/Microsoft.DevTestLab/locations/{locationName}/operations/{name}` is a generic endpoint to query the status of any long running operations in the dev-test lab RP. @ajay-arora  there are bunch of operations marked as `x-ms-long-running-operation,` i guess they return `Azure-AsyncOperation` header with url to poll the status. Based on our discussion format of this header url is `/subscriptions/{subscriptionId}/providers/Microsoft.DevTestLab/locations/{locationName}/operations/{name}`. Right?

Reply:
Correct